EIPS/eip-8300.md

```diff
@@ -0,0 +1,91 @@
+---
+eip: 8300
```

Comment:
```suggestion
eip: 7650
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.